media-gfx/openscad/openscad-2019.05.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+# 2019.05
+#MY_VER=$(ver_cut 1-2) # version component
```

Comment:
any reason you keep those commented lines?

Reply:
No, just didn't clean up the file properly.